react-time-track/src/index.js

```diff
@@ -1,29 +1,92 @@
-import React from "react";
+/** @jsx jsx */
+import React, { useState } from "react";
```

Comment:
Hi, what do you think about using only one of them instead of declaring both?, I mean If it was only React, you would use in some commands React.useState, otherwise it would only be useState.

Reply:
Thanks, I have deleted React and left only useState